apps/internal/oauth/ops/ops.go

```diff
@@ -17,22 +17,22 @@ Usage is simple:
 package ops
 
 import (
-	"net/http"
-
 	"github.com/AzureAD/microsoft-authentication-library-for-go/apps/internal/oauth/ops/accesstokens"
 	"github.com/AzureAD/microsoft-authentication-library-for-go/apps/internal/oauth/ops/authority"
 	"github.com/AzureAD/microsoft-authentication-library-for-go/apps/internal/oauth/ops/internal/comm"
 	"github.com/AzureAD/microsoft-authentication-library-for-go/apps/internal/oauth/ops/wstrust"
 )
 
+type HTTPClient = comm.HTTPClient
```

Comment:
I *think* you might need to copy the comments.  I can't remember if type equivalence copies them for the godoc or not.  Might be worth checking.

Reply:
I think you're right, if anything it will shut up the linter in vs code.